.github/workflows/compile.yml

```
@@ -25,6 +25,7 @@ jobs:
         os:
         - macOS-13
         - macOS-14
+        - macos-15 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)